### ISSUE TITLE
QA issues arising from popbio data load for the beta release

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/EdaLoadingIssues#83]
Currently testing on https://feature.vectorbase.org/vectorbase.feature/app/workspace/maps/DS_480c976ef9/new

- [x] Study name is `Mega test 1` - could be `Field-based vector population studies` ?
- [x] featured variable doesn't seem to work - should we fake it in the database for OBI_0001909? ('species')
Here is the annotation in the popbio.owl file
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/6d4da54f-e77d-4d47-969b-f94684542321)
But here is the study metadata from the service
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/3b91f2cb-4dca-4478-9058-9f5d62a00075)

- [x] isTemporal annotations also missing (impact: no variables suitable for timeline plot!)
Owl file annotation:
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/8e3bb590-e6f7-45f4-82f0-8040167ca1ac)
Study metadata:
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/34890a4c-7693-45d9-b2ba-6d79bd03ea2d)

Let's take a look at those annotations in more detail:
owl file is_featured is `EUPATH_0001012`, is_temporal is `EUPATH_0001015`

- [x] character encodings in place names (and possibly also study titles)
It's not a client thing. The back end is returning: `H? Chi Minh (Vietnam)`
The database (e.g. `ATTRIBUTES_2018_ITOKAWA_ALBOPICTUS_COLLECTIONSITE`) looks like it's storing the data OK. Maybe the service (no flat files currently active on feature.vectorbase.org) isn't handling them?
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/ee05873e-bd96-4741-9062-3fe0355255cb)

- [x] study details page - contacts and description are still placeholders

- [ ] lat/long columns are missing from downloads (tested on https://feature.vectorbase.org during release day window)
       - this is odd because lat/long are downloadable from SCORE in regular clinepi EDA 
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/2d141c5f-8e8c-4006-a6c3-beeba98047a4)

**Comment by bobular** [comment]
Featured and temporal annotations are working.  Default variable in markers is still `contact` which is likely a client bug. Made a client ticket: https://github.com/VEuPathDB/web-monorepo/issues/486

**Comment by bobular** [comment]
Mysterious entity counts

TL;DR: these are records with no lat/long
Solution -> https://github.com/VEuPathDB/web-monorepo/issues/503


megastudy
bubble markers with "mortality rate" variable
filter on all values for "mortality rate"
zoom out as far as you can

Type | # Insecticide assays
---|---
All | 19,710
Filtered | 17,284
View | 16,786

Why is the "View" number so much smaller than filtered?

How about using a different variable for the markers?
donut markers with Sample type (100% coverage)

(Still filtering on "having values mortality rate")

Type | # Samples
---|---
All | 1,837,174
Filtered | 10,874
View | 10,694

Still some missing. Could it be missing lat/long?

Well I can't check that in subsetting because they are hidden and they are not in downloads - see above (should we at least have a "has_latlong" boolean variable?)

But we can filter on "have values for country" which can only be determined if there is lat/long at load time.

Ah... If we do that then there are no more samples missing from view:

Type | # Samples
---|---
All | 1,837,174
Filtered | 10,662
View | 10,662

**Comment by bobular** [comment]
![image](https://github.com/VEuPathDB/EdaLoadingIssues/assets/308639/661ea368-6bb1-47d1-b2eb-76e99759d282)
Geolocation provenance has non-informative values such as `geolocation obfuscation information`
